packages/instant/src/components/secondary_button.tsx

```diff
@@ -0,0 +1,28 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { ColorOption } from '../style/theme';
+
+import { Button, ButtonProps } from './ui/button';
+import { Text } from './ui/text';
+
+export interface SecondaryButtonProps extends ButtonProps {
+    text: string;
+}
+
+export const SecondaryButton: React.StatelessComponent<SecondaryButtonProps> = props => {
+    const buttonProps = _.omit(props, 'text');
```

Comment:
I usually do `{ text, ...rest }` but this works too. Just a heads up.

Reply:
Actually, in a future branch I am going to have this to just render `children` instead of a `text` prop, so this will be changing shortly in the future, going to leave as-is in this PR